ruby.html.markdown

```diff
@@ -124,6 +125,11 @@ puts "I'm printing!"
 print "I'm printing!"
 #=> I'm printing! => nill
 
+#You can use return in ruby
```

Comment:
This is an odd place for this explanation. I believe there's already a line about this in the section about functions.